data/pt_PT/system/analyzer/resources/messages_pt_PT.properties

```diff
@@ -1173,69 +1173,12 @@ dlgPageSetupPdfPageFormatA5=A5
 dlgPageSetupPdfPageFormatB4=B4
 dlgPageSetupPdfPageFormatB5=B5
 linkDlgBrowseButton=Explorar...
-actionAddLevel=Adding Level %{0}<TRANSLATE ME>
-actionAddMeasure=Adding Measure %{0}<TRANSLATE ME>
-actionMoveLevel=Moving Level %{0}<TRANSLATE ME>
-actionMoveMeasure=Moving Measure %{0}<TRANSLATE ME>
-actionRemoveLevel=Removing Level %{0}<TRANSLATE ME>
-actionRemoveMeasure=Removing Measure %{0}<TRANSLATE ME>
-popupMenuExportXLSX=To Excel Workbook...<TRANSLATE ME>
-dlgCalcNumClear=Clear<TRANSLATE ME>
-selectSchemaTooltip=Help on selecting data sources<TRANSLATE ME>
-actionUpdateMeasure=Update measure %{0}<TRANSLATE ME>
```

Comment:
You are not supposed to remove keys. If you do not want to translate them right now, leave them with the <TRANSLATE ME> tag.
Please restore the removed lines.